src/config.c

```diff
@@ -2128,6 +2129,20 @@ const idclass_t config_class = {
       .group  = 1
     },
     {
+      .type   = PT_BOOL,
+      .id     = "proxy",
+      .name   = N_("Use PROXY protocol & X-Forwarded-For"),
+      .desc   = N_("PROXY protocol is an extension for support incoming "
+                   "TCP connections from a remote server (like a firewall) "
+                   "sending the original IP address of the client. "
+                   "The HTTP header 'X-Forwarded-For' do the same with "
+                   "HTTP connections. Both enable tunneled connections."
+                   "This option should be disabled for standard usage."),
```

Comment:
Hi,

It could have IPv6 support, as the PROXY protocol supports it. Also, it could have PROXY v2 (binary) support too. However, we prefer to only integrate the most simple option at time.

Feel free to add "(Note: only IPv4 support at time)".